package.json

```diff
@@ -35,7 +35,9 @@
     "build-storybook": "build-storybook",
     "jest:integration": "jest -c jest-snapshot.config.js --verbose ./tst/snapshots/*",
     "test:snapshots-path": "jest -c jest-snapshot.config.js --verbose",
-    "test:snapshots": "start-server-and-test start http-get://localhost:9009 jest:integration"
+    "test:snapshots": "start-server-and-test start http-get://localhost:9009 jest:integration",
+    "test:integration": "cd ./integration && npm run test",
```

Comment:
As discussed yesterday, can we move this to under the integ folder?

Reply:
Updated.